cypress/integration/search_spec.js

```diff
@@ -0,0 +1,130 @@
+describe('Search', function() {
+  beforeEach(function() {
+    cy.visit('/')
+    cy.get('#filter-bar').as('filterBar')
+    cy.get('#movie-list').as('movieList')
+  })
+
+  it('has "Sort By"', function() {
+    cy.get('@filterBar').contains('Sort By')
+  })
+
+  it('has input', function() {
+    cy.get('@filterBar').find('input')
+  })
+
+  it('has Search button', function() {
+    cy
+      .get('@filterBar')
+      .contains('Search')
+      .should('match', 'button')
+  })
+
+  it('has 10 default results', function() {
+    cy
+      .get('@movieList')
+      .find('.movie-row')
+      .should('have.length', 10)
+  })
+
+  describe('when sorted', function() {
+    context('by Year', function() {
+      beforeEach(function() {
+        cy
+          .get('@filterBar')
+          .contains('Sort By')
+          .click()
+        cy
+          .get('@filterBar')
+          .contains('Year')
+          .click()
+      })
+
+      it('sorts earliest first', function() {
+        cy
+          .get('@movieList')
+          .first('.movie-row')
+          .contains('All Quiet on the Western Front')
+
+        cy.screenshot()
+      })
+    })
+
+    context('by Movie runtime', function() {
+      beforeEach(function() {
+        cy
+          .get('@filterBar')
+          .contains('Sort By')
+          .click()
+        cy
+          .get('@filterBar')
+          .contains('Movie runtime')
+          .click()
+      })
+
+      it('sorts shortest first', function() {
+        cy
+          .get('@movieList')
+          .first('.movie-row')
+          .contains('Shogun')
```

Comment:
Again, the preference would be to aggregate the movie runtimes, sort, & compare the sorted list to the movie listing, but my preference would be to adjust the markup to make that easier with less boilerplate.